challenge_3/python/lex32pem/challenge_3.py

```diff
@@ -0,0 +1,58 @@
+# Challenge # 3
+# 2017 Programming Challenge @ Slack
+# 
+# Majority Element
+# 
+# Premise
+# 
+# Given an array of size n, find the majority element. The majority element is the element
+# that appears more than ⌊ n/2 ⌋ times.
+# 
+# You may assume that the array is non-empty and the majority element always exist
+# in the array.
+# 
+# for example, given:
+# 
+# array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7]
+# 
+# your program should return 7
+# 
+# Language: Python 3.6
+
+def find_Max(u):
+    dictMax = {}                            # empty dictionary
+    max = ('', 0)                           # (max element, max occurrences)
```

Comment:
careful with this one, I think its stupid but python will allow you to use a variable name that is also a function which can lead to unclear and sometimes unpredictable code